### DISCUSSION TITLE
Model not Found

### DISCUSSION THREAD

**Discussion by nicolecosta** [nomic-ai/pygpt4all#31] [Q&A]
Hey, I was following the pyllama installation and usage ([https://github.com/nomic-ai/pyllamacpp]) and got the following error: Exception: File [./models/gpt4all-model.bin]) not found!

Can anyone help me figure out what is going on?

**Comment by AlyceOsbourne** [comment]
the path is assumed to be relative to the ran file. so
```
script.py
models/
    gpt4all-model.bin
```

**Reply by nicolecosta** [reply]
Thank you for replying, however I'm not sure I understood how to fix the problem

**Comment by absadiki** [comment]
@nicolecosta, the error said that the script did not find the model you are pointing to!
you need to download the model and specify its path in the script!
